src/s25update.cpp

```diff
@@ -191,8 +195,13 @@ bool DoDownloadFile(const std::string& url, std::string* to, const bfs::path& pa
     if(progress)
     {
         curl_easy_setopt(curl_handle, CURLOPT_NOPROGRESS, 0L);
+	#if LIBCURL_VERSION_NUM >= 0x011DF0
```

Comment:
wrong indentation, preprocessor arguments should be "all left" always

Reply:
Fixed. Thanks for the Input.